.circleci/config.yml

```diff
@@ -75,16 +75,12 @@ jobs:
 
    - run:
        name: install node conductor
-       command: nix-shell --run hc-conductor-node-install
+       command: nix-shell --run hc-conductor-rust-install
```

Comment:
instead of replacing this, but still calling it 'install node conductor' I think it'd be better to just remove this command, and point whatever was calling this command to the existing job

Reply:
actually I see that you replaced it already in `holonix/app-spec/src/test.nix` so just remove this command instead